insights/core/ls_parser.py

```diff
@@ -0,0 +1,135 @@
+import logging
```

Comment:
@csams it would be really nice if you could add some docstrings to this module since its the core of the ls command parsing code.

Reply:
Other than the docs, the changes look good to me.

Reply:
@bfahr I need to add handling for human readable sizes.